docs/auth.md

```diff
@@ -305,7 +305,7 @@ specified in the `Run`.
 1. Execute the `Run`:
 
    ```shell
-   kubectl apply --filename secret.yaml serviceaccount.yaml run.yaml
+   kubectl apply --filename secret.yaml,serviceaccount.yaml,run.yaml
```

Comment:
I don't think that's valid either. `--filename` needs to be repeated

```bash
kubectl apply -f ./my1.yaml -f ./my2.yaml # short flags
```

Reply:
it worked when i tried

```
kc apply -f https://raw.githubusercontent.com/tektoncd/pipeline/main/config/200-serviceaccount.yaml,https://raw.githubusercontent.com/tektoncd/pipeline/main/config/200-role.yaml,https://raw.githubusercontent.com/tektoncd/pipeline/main/config/201-rolebinding.yaml
```

kubectl docs: 
```
filename      f      []     that contains the configuration to apply
```
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply